bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
Could have seen that one coming ;-)

And I do agree it is a though one*

## Concepts

Okay we have the following concepts:

**i18n**: 
> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.

[Source](https://www.w3.org/International/questions/qa-i18n)

**Locale** 
> A Locale object represents a specific geographical, political, or cultural region. 

[Source](https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html)

**Location** ;-)
> An absolute location is designated using a specific pairing of latitude and longitude in a Cartesian coordinate grid

[Source](https://en.wikipedia.org/wiki/Location_(geography))

## First Option

Given that maybe providing a Location is a form of *i18n* but its not a locale, so for example

`i18n.InternationalizationProvider`

## Alternative

An alternative could be: **globalization** 

> Some companies, like IBM and Sun Microsystems, use the term "globalization", g11n, for the combination of internationalization and localization.[6] Also known as "glocalization" (a portmanteau of globalization and localization).

[source](https://en.wikipedia.org/wiki/Internationalization_and_localization)

globalization.GlobalizationProvider

\* Maybe that actually means they are not closely enough related, but I can understand this choice.